x-pack/plugins/ml/public/data_frame/pages/job_management/components/job_list/use_refresh_interval.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+
+import { timefilter } from 'ui/timefilter';
+
+import {
+  DEFAULT_REFRESH_INTERVAL_MS,
+  MINIMUM_REFRESH_INTERVAL_MS,
+} from '../../../../../../common/constants/jobs_list';
+
+import { GetJobs } from './job_service/get_jobs';
+
+export const useRefreshInterval = (
```

Comment:
discussed already, but adding here for the sake of it.
As this is so similar to the interval checking code in the jobs list, it would be nice to create a shared function which they can both use.
At some point in the future...